templates/designateapi/bin/bootstrap.sh

```diff
@@ -15,6 +15,7 @@
 # under the License.
 set -ex
 
-OPTS="--config-file /etc/designate/designate.conf"
-designate-manage database ${OPTS} upgrade head
+# Note(dkehn) - this maybe need to change back to upgrade head?
+# designate-manage database upgrade head
+designate-manage database sync
```

Comment:
Curious why this needs to be a shell script. Couldn't this be run by the golang directly?